sanitizer/scrape.go

```diff
@@ -0,0 +1,32 @@
+package sanitizer
+
+import (
+	//"github.com/iBatStat/extractor/model"
+	"fmt"
+	gotes "github.com/otiai10/gosseract"
+	"log"
+)
+
+var (
+	ocr *gotes.Client
+)
+
+func testOCR(filePath string) {
+	out, err := ocr.Src(filePath).Out()
+	if err != nil {
+		log.Fatal(err)
+		return
+	}
+	fmt.Println(fmt.Sprintf("****** Extracted data is *********\n%s", out))
+}
+func init() {
+	var err error
+	ocr, err = gotes.NewClient()
```

Comment:
if this craps out, lets return the error... we need to terminate the app since its useless without it 

I guess the "init" should be an exposed Init and invoked from main... what say?